A3ServerTool/Attributes/ConfigProperty.cs

```diff
@@ -13,6 +13,11 @@ internal sealed class ConfigProperty : Attribute
         /// </summary>
         public string PropertyName { get; set; }
 
+        /// <summary>
+        ///  Parent class while parsing.
+        /// </summary>
+        public string Parent { get; set; }
```

Comment:
Should this be an array/list instead to support multiple parent classes?

Reply:
Yes,. This allow us to get rid of WrappingClass attribute and basically unify everything under  ConfigProperty.

```
class DifficultyPresets
{
	class CustomDifficulty
	{
		class Options
		{
			reducedDamage=0;
                        //rest of .Arma3Profile file
                };
        };
};
```